party.supertux.supertuxparty.json

```diff
@@ -50,7 +50,6 @@
 	],
 	"finish-args": [
 		"--socket=x11",
-		"--socket=wayland",
```

Comment:
Is there a particular reason to remove this?

Reply:
Try removing the X11 socket, or instead of `x11` using the proper `fallback-x11`. It doesn't run saying it can't open the X11 display. Which mean that it doesn't support wayland so the socket has no reason to be.

(Godot doesn't seem to support wayland)

Reply:
While this is true, there seem to be issues with removing this permission on some wayland configurations. See https://gitlab.com/SuperTuxParty/SuperTuxParty/-/issues/146

So I'd prefer to keep this permission